diff --git a/neofetch b/neofetch
index 48b96d215..39192937e 100755
--- a/neofetch
+++ b/neofetch
@@ -999,6 +999,13 @@ get_distro() {
                     *) distro="OS Elbrus $(< /etc/mcst_version)"
                 esac
 
+            elif [[ -f /etc/NIXOS ]]; then
+                case $distro_shorthand in
+                    on) distro="NwixOwOS $(nixos-version | awk '{print substr($1,0,5),$2}')" ;;
+                    tiny) distro="NwixOwOS" ;;
+                    *) distro="NwixOwOS $(nixos-version)" ;;
+                esac
+
             elif type -p pveversion >/dev/null; then
                 case $distro_shorthand in
                     on|tiny) distro="Proxmox VE" ;;

